author: Andres Salomon <dilinger@debian.org>
description: disable kMojoIpcz to fix trap runtime error on i386

See https://bugs.debian.org/1034406

--- a/mojo/core/embedder/features.cc
+++ b/mojo/core/embedder/features.cc
@@ -37,7 +37,7 @@ BASE_FEATURE(kMojoInlineMessagePayloads,
 #if BUILDFLAG(IS_CHROMEOS)
 BASE_FEATURE(kMojoIpcz, "MojoIpcz", base::FEATURE_DISABLED_BY_DEFAULT);
 #else
-BASE_FEATURE(kMojoIpcz, "MojoIpcz", base::FEATURE_ENABLED_BY_DEFAULT);
+BASE_FEATURE(kMojoIpcz, "MojoIpcz", base::FEATURE_DISABLED_BY_DEFAULT);
 #endif
 #endif  // BUILDFLAG(MOJO_SUPPORT_LEGACY_CORE)
 
--- a/mojo/core/embedder/embedder.cc
+++ b/mojo/core/embedder/embedder.cc
@@ -188,6 +188,8 @@ scoped_refptr<base::SingleThreadTaskRunn
 }
 
 bool IsMojoIpczEnabled() {
+  return false;
+/*
 #if !BUILDFLAG(MOJO_SUPPORT_LEGACY_CORE)
   return true;
 #else
@@ -205,6 +207,7 @@ bool IsMojoIpczEnabled() {
   DCHECK_EQ(enabled, enabled_on_first_call);
   return enabled;
 #endif
+*/
 }
 
 void InstallMojoIpczBaseSharedMemoryHooks() {
